### PR TITLE
corrections of mainnav caption

### PR DESCRIPTION
**Pull request by dev7ch**

### CODE REVIEW

File: modules/admin/src/resources/scss/components/_mainnav.scss [hunk 1]
```diff
@@ -97,7 +97,7 @@
 
 .mainnav-link {
     border-radius: 5px;
-    color: map-get($grays, 400);
+    color: map-get($grays, 400) !important;
```
Comment (nadar) [inline modules/admin/src/resources/scss/components/_mainnav.scss]: !important but in the same class:

File: modules/admin/src/resources/scss/components/_mainnav.scss [hunk 2]
```diff
@@ -208,6 +208,8 @@
     min-width: 24px;
     width: 100%;
     max-width: 25px;
+    
+    color: map-get($grays, 400);
```
Comment (nadar) [inline modules/admin/src/resources/scss/components/_mainnav.scss]: in the same class?

Comment (dev7ch) [reply modules/admin/src/resources/scss/components/_mainnav.scss]: its not the same class '.mainnav-link' and '.mainnav-image-icon'

**Comment by dev7ch** [comment]
why is Travis failing?

**Pull request merged**